README.md

```diff
@@ -114,7 +114,7 @@ Table of Contents
     * Full, detailed list - https://www.oracle.com/cloud/free/
 
   * [IBM Cloud](https://www.ibm.com/cloud/free/)
-    * Kubernetes - Free cluster with 1 worker node
+    * Kubernetes - Free cluster with 1 worker node for 30 days
```

Comment:
then we can remove it, free for 30 days doesnt qualify for this list

thanks!

Reply:
Done